projects/SlackClone/README.md

```diff
@@ -38,11 +38,8 @@ cd react-native-samples/projects/slack-clone
 # Install npm dependencies
 yarn
 
-# Link the assets
-npx react-native link
-
 # Install pod dependencies
-npx pod-install
+cd ios && npx pod-install && cd ..
```

Comment:
you don't need to change directories with npx https://reactnative.dev/docs/libraries#linking-native-code-on-ios